src/NetAppFiles/NetAppFiles/help/Get-AzNetAppFilesAccount.md

```diff
@@ -0,0 +1,155 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.NetAppFiles.dll-Help.xml
+Module Name: Az.NetAppFiles
+online version: https://docs.microsoft.com/en-us/powershell/module/az.netappfiles/get-aznetappfilesaccount
+schema: 2.0.0
+---
+
+# Get-AzNetAppFilesAccount
+
+## SYNOPSIS
+Gets details of an Azure NetApp Files (ANF) account.
+
+## SYNTAX
+
+### ByFieldsParameterSet (Default)
+```
+Get-AzNetAppFilesAccount -ResourceGroupName <String> [-Name <String> | -AccountName <String>]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ByResourceIdParameterSet
+```
+Get-AzNetAppFilesAccount -ResourceId <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzNetAppFilesAccount** cmdlet gets details of an ANF account.
+
+## EXAMPLES
+
+### Example 1: Get an ANF account
+```
+PS C:\>Get-AzNetAppFilesAccount -ResourceGroupName "MyRG" -Name "MyAnfAccount"
+```
+
+This command gets the account named MyAnfAccount.
+
+Output:
```

Comment:
@leonardbf for all examples, please move the corresponding output into the code block with the example commands (this will preserve the formatting of the output)

Reply:
I originally had these split but only one code block is allowed. Combining it works for me. Done.